src/ucs/type/serialize.h

```diff
@@ -17,14 +17,29 @@
  *
  * @param _iter   Pointer to a pointer, representing the current element.
  * @param _type   Type of the current element.
+ * @param _size   Current element size.
```

Comment:
is this an offset to advance the pointer? If this was a current element size, `ucs_serialize_next` would be enough, right?

Reply:
updated documentation